### PR TITLE
document workflow log files

### PR DESCRIPTION
**Pull request by oliver-sanders**
**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.

### CODE REVIEW

File: src/user-guide/running-workflows/scheduler-log-files.rst [hunk 1]
```diff
@@ -1,7 +1,112 @@
 .. _Scheduler Logs:
+.. _user-guide.log_files:
 
-Scheduler Logs
---------------
+Workflow Logs
+=============
+
+Cylc produces a number of log files which record information about the
+wortkflow, how it was installed, what tasks have run, their outcomes and more.
```
Comment (wxtim) [inline src/user-guide/running-workflows/scheduler-log-files.rst:8]:
```suggestion
workflow, how it was installed, what tasks have run, their outcomes and more.
```

Wortflow? Is that a brewing term?

Comment (wxtim) [inline src/user-guide/running-workflows/scheduler-log-files.rst:8]: I think it's worth ordering this sentence by the order in which Cylc logs stuff.
```suggestion
Cylc produces log files which record information how the workflow was
installed, configured when started or changed, and about tasks run.
```

**Review by wxtim — COMMENTED** [review]
If you disagree with me about the one comment I've made I'll approve, but I think it's worth considering.

File: src/user-guide/running-workflows/scheduler-log-files.rst (hunk 1 above)
Comment (oliver-sanders) [reply src/user-guide/running-workflows/scheduler-log-files.rst:8]: Missing an "about" between "information" and "how"

Comment (oliver-sanders) [reply src/user-guide/running-workflows/scheduler-log-files.rst:8]: Sorry, "and about tasks run" doesn't make sense.

"and the tasks that have run and their outcomes"?

Comment (wxtim) [reply src/user-guide/running-workflows/scheduler-log-files.rst:8]: Have edited my proposed change. I agree.

**Review by wxtim — APPROVED** [review]

**Comment by oliver-sanders** [comment]
(CI failure is a linkcheck issue, fixed by #761)

**Comment by oliver-sanders** [comment]
* Rebased
* Placated spellcheck
* Re-ran CI post #761 merge

**Pull request merged**